FirstProject/assets/evaluation.less

```diff
@@ -0,0 +1,64 @@
+//Evaluation less file
+
+.row {
+
+  .navigation_bar {
+    border-bottom: 2px solid darkgrey;
+    padding-top: 12px;
+    padding-bottom: 14px;
+  }
+
+  .logo_image {
```

Comment:
I removed nesting from evaluation.less.
I made a new push now with this file.

thanks 👍